torch/_inductor/codecache.py

```diff
@@ -614,22 +629,32 @@ def get_include_and_linking_paths(
         # to do to enable OMP build on darwin where PyTorch is built with IOMP
         # and we need a way to link to what PyTorch links.
         ipaths = cpp_extension.include_paths(cuda) + [sysconfig.get_path("include")]
+        if aot_mode:
+            ipaths += [os.path.join(_TORCH_PATH, "_inductor", "aot_cpp_include")]
         lpaths = cpp_extension.library_paths(cuda) + [
             sysconfig.get_config_var("LIBDIR")
         ]
-        libs = ["c10", "torch", "torch_cpu", "torch_python"]
-        libs += ["gomp"]
+        libs = []
+        # No need to manually specify libraries in fbcode.
+        if not config.is_fbcode():
+            libs += ["c10", "torch", "torch_cpu", "torch_python"]
+            libs += ["gomp"]
         macros = vec_isa.build_macro()
         if macros:
             macros = f"-D{macros}"
         if cuda:
-            libs += ["c10_cuda", "cuda", "torch_cuda"]
+            if config.is_fbcode():
+                libs += ["cuda"]
+            else:
+                libs += ["c10_cuda", "cuda", "torch_cuda"]
     else:
         # Note - this is effectively a header only inclusion. Usage of some header files may result in
         # symbol not found, if those header files require a library.
         # For those cases, include the lpath and libs command as we do for pytorch above.
         # This approach allows us to only pay for what we use.
         ipaths = cpp_extension.include_paths(cuda) + [sysconfig.get_path("include")]
+        if aot_mode:
+            ipaths += [os.path.join(_INDUCTOR_PATH, "aot_cpp_include")]
```

Comment:
This else branch is not tested, that's why _INDUCTOR_PATH is not defined but not caught. Honestly we don't care about this else branch, so probably ok to drop the change here.

Reply:
Done. Thanks.